server/plugin.go

```diff
@@ -43,20 +43,18 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	if !strings.HasPrefix(args.Command, "/"+CommandTrigger) {
 		return &model.CommandResponse{
 			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-			Text:         fmt.Sprintf("Unknown command: " + args.Command),
+			Text:         fmt.Sprintf("Unknown command: %s", args.Command),
 		}, nil
 	}
 
-	m, err := p.buildAnalyticMsg()
-	if err != nil {
-		return nil, &model.AppError{Message: err.Error()}
+	if err := p.sendAnalytics([]string{args.ChannelId}); err != nil {
+		return &model.CommandResponse{
+			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
+			Text:         fmt.Sprintf("Error: %s", err.Error()),
```

Comment:
3/5 that you should not expose the error message to the user and instead log it.